core/templates/domain/topic/TopicObjectFactory.ts

```diff
@@ -541,6 +558,31 @@ export class Topic {
     return this._uncategorizedSkillSummaries.slice();
   }
 
+  getDiagnosticTestSkillSummaries(): ShortSkillSummary[] {
+    return this._skillsForDiagnosticTest.slice();
+  }
+
+  setDiagnosticTestSkillSummaries(
+      skillsForDiagnosticTest: ShortSkillSummary[]): void {
+    this._skillsForDiagnosticTest = skillsForDiagnosticTest;
+  }
+
+  getSkillSummariesThatAreNotInDiagnosticTest(): ShortSkillSummary[] {
```

Comment:
Hm ... yeah, I think I lean more towards changing _skillsForDiagnosticTest to _skillSummariesForDiagnosticTest. It's weird to have different names for the same thing.

Reply:
Updated